themes/edx.org/lms/templates/footer.html

```diff
@@ -18,83 +19,269 @@
-          <p class="copyright">${_(
-          u"\u00A9 2012\u2013{year} edX Inc.  All rights reserved except where noted.  "
-          u"EdX, Open edX and the edX and Open edX logos are registered trademarks "
-          u"or trademarks of edX Inc."
-          ).format(year=datetime.datetime.now().year)}
-          ${u" | {icp}".format(icp=getattr(settings,'ICP_LICENSE')) if getattr(settings,'ICP_LICENSE',False) else ""}
-	  </p>
-
-          ## The Open edX link may be hidden when this view is served
-          ## through an API to partner sites (such as marketing sites or blogs),
-          ## which are not technically powered by Open edX.
-          % if not hide_openedx_link:
-          <div class="openedx-link">
-            <a href="${footer['openedx_link']['url']}" title="${footer['openedx_link']['title']}">
-              <img alt="${footer['openedx_link']['title']}" src="${footer['openedx_link']['image']}" width="140">
-            </a>
-          </div>
-          % endif
-      </div>
-
-      <div class="external-links">
-        <ul class="social-media-links">
-            % for link in footer['social_links']:
-            <li class="list-item">
-                <a href="${link['url']}" class="sm-link external" title="${link['title']}" rel="noreferrer">
-                    <span class="icon fa ${link['icon-class']}" aria-hidden="true"></span>
-                    <span class="sr">${link['action']}</span>
+<div class="container">
+    <div class="row all-screens"><!-- row -->
+        <div class="col-lg-2 col-md-12 col-sm-3 col-xs-10">
+            <div class="edx-footer-logo">
+                <a href="${marketing_link('ROOT')}">
+                    <img alt="${_('edX Home Page')}" src="${footer['logo_image']}">
                 </a>
-            </li>
-            % endfor
-        </ul>
-
-        <ul class="mobile-app-links">
-            % for link in footer['mobile_links']:
-            <li class="list-item">
-                <a href="${link['url']}" class="app-link external">
-                    <img alt="${link['title']}" src="${link['image']}">
+            </div>
+
+            % if context.get('include_language_selector', footer_language_selector_is_enabled()):
+            <%include file="${static.get_template_path('widgets/footer-language-selector.html')}"/>
+            % endif
+        </div>
+        <div class="col-lg-10 col-md-12 col-sm-9 col-xs-12">
+             <div class="row">
+
+                 <div class="col-md-3 col-sm-4 col-xs-12">
+                    <div class="row" >
+                        <div class="col-lg-12 col-md-12 col-sm-12" >
+                          <ul class="list-unstyled clear-margins left-offset" >
+                            <li class="widget-container widget_nav_menu">
+                              <h3>edX</h3>
+                                  <ul>
+                                      % for link in footer["edx_links"]:
```

Comment:
As mentioned before, this should not be being provided by the API in this way.

Reply:
Are we talking about the name here again?